public/sass/elements/forms/_form-multiple-choice.scss

```diff
@@ -8,19 +8,15 @@
   clear: left;
   position: relative;
 
-  padding: (8px $gutter-one-third 9px 50px);
+  padding: 0 0 0 38px;
   margin-bottom: $gutter-one-third;
 
-  cursor: pointer; // Encourage clicking on the label
-
    // remove 300ms pause on mobile
   -ms-touch-action: manipulation;
   touch-action: manipulation;
```

Comment:
Although if we want to test and make that change as a follow-up I’m happy with that too.